packages/jest-message-util/src/index.js

```diff
@@ -277,14 +278,15 @@ export const formatStackTrace = (
   }
 
   const stacktrace = lines
+    .filter(Boolean)
```

Comment:
Stack traces shouldn't contain blank lines (except for one above it added below)